benchmarks/trace/generate.go

```diff
@@ -0,0 +1,161 @@
+package trace
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"math"
+	"net"
+	"os"
+	"path/filepath"
+	"testing"
+	"text/template"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+
+	"go.opencensus.io/benchmarks/defs"
+	"go.opencensus.io/plugin/ocgrpc"
+
+	"github.com/odeke-em/cli-spinner"
```

Comment:
Don't import third party dependencies.

Reply:
That's the spinner though, but sure, I can remove it.